static/src/elements/santa-santa.js

```diff
@@ -52,21 +52,32 @@ common.preload.assets(...urls.map((raw) => `${assetRoot}/${raw}`));
 
 
 class SantaSantaElement extends LitElement {
-  static get styles() { return [styles]; }
+  static get styles() {return [styles];}
 
   static get properties() {
     return {
       heading: {type: Number},
+      locationLabel: {type: String},
       stop: {type: Boolean},
     };
   }
 
   render() {
     const dir = dirForHeading(this.heading);
     const mode = Math.floor(Math.abs(this.heading) % 2);  // could be decimal
+
+    let label = _msg`santasearch_character_santa`;
+    if (this.locationLabel) {
+      if (this.stop) {
+        label = label + ` is at ${this.locationLabel}`;
+      } else {
+        label = label + ` is heading to ${this.locationLabel}`;
+      }
+    }
+
```

Comment:
Hm. I think this might end up being awkward for languages other than English. Might need to figure out how to dynamically generate this string in a way that works for multiple languages